src/scan-binary.sh

```diff
@@ -36,3 +59,13 @@ bash <(curl -s -L $DETECT_URL_PATH) \
      --detect.project.name=${PROJECT} \
      --detect.project.version.name=${VERSION} \
      --detect.code.location.name=${PROJECT}_${VERSION}_${SUFFIX}_code \
+
+if [ "$DETECT_SERIAL_MODE" = "true" ] || [ "$DETECT_SERIAL_MODE" = "TRUE" ]; then
```

Comment:
Only if user's specifying the serial mode poll the server until the scan is completed server-side.